cypress/utils/data_utils.ts

```diff
@@ -15,3 +15,7 @@ export function getBusinessServiceName(): string {
 export function getBusinessServiceDescription(): string {
```

Comment:
since we are creating these methods in data_utils now . We can create methods like getDescription() which can be used in all model classes .

Reply:
I completely agree :+1: